ui-components/src/actions/Button.scss

```diff
@@ -145,6 +145,12 @@ button {
   }
 }
 
+button {
+  &.no-underline {
+    text-decoration: none !important;
```

Comment:
After we paired, were you able to see if this could be removed with no functionality change since we have `no-underline` below?

Reply:
@kramduckner ^

Reply:
@emilyjablonski yup! if we don't include that class we get an underline attached to the button
![Screen Shot 2022-10-28 at 5 23 06 PM](https://user-images.githubusercontent.com/5691923/198749981-61768506-94de-4c84-9cf2-dbf3d6e98789.png)